flow360/component/simulation/models/surface_models.py

```diff
@@ -165,7 +165,7 @@ class Wall(BoundaryBase):
         None, description="Prescribe a tangential velocity on the wall."
     )
     heat_spec: Optional[Union[HeatFlux, Temperature]] = pd.Field(
-        None,
+        HeatFlux("0"),
```

Comment:
Can we not use expression here? Why do we use expression when we are trying to set a constant value?

Reply:
Sure, use a constant value now.